src/utilities/make_state_vector_file.py

```diff
@@ -46,49 +46,60 @@ def check_nested_grid_compatibility(lat_min, lat_max, lon_min, lon_max, land_cov
 
 def make_state_vector_file(
     land_cover_pth,
+    hemco_diag_pth,
     save_pth,
     lat_min,
     lat_max,
     lon_min,
     lon_max,
     buffer_deg=5,
     land_threshold=0.25,
+    emis_threshold=1e-12,
     k_buffer_clust=8,
 ):
     """
     Generates the state vector file for an analytical inversion.
 
     Arguments
         land_cover_pth [str]   : Path to land cover file
+        hemco_diag_pth [str]   : Path to initial HEMCO diagnostics file
         save_pth       [str]   : Where to save the state vector file
         lat_min        [float] : Minimum latitude
         lat_max        [float] : Maximum latitude
         lon_min        [float] : Minimum longitude
         lon_max        [float] : Maximum longitude
         buffer_deg     [float] : Width of k-means buffer area in degrees
         land_threshold [float] : Minimum land fraction to include pixel as a state vector element
+        emis_threshold [float] : Minimum emission to include offshore pixel as a state vector element
         k_buffer_clust [int]   : Number of buffer clusters for k-means
 
     Returns
         ds_statevector []     : xarray dataset containing state vector field formatted for HEMCO
 
     Notes
         - Land cover file looks like 'GEOSFP.20200101.CN.025x03125.NA.nc'
+        - HEMCO diags file needs to be global, is used to include offshore emissions in state vector
+        - Land cover file and HEMCO diags file need to have the same grid
```

Comment:
Just looking at the comment -- does this still work with the .5 degree option?

Reply:
Yes it should still work -- will clarify the comment.